### PR TITLE
Some loc fixes

### PR DESCRIPTION
**Pull request by FlashVanMaster**

### CODE REVIEW

**Comment by FlashVanMaster** [comment]
New and old abilities

File: LongWarOfTheChosen/Localization/LW_FactionBalance/LW_FactionBalance.rus [hunk 1]
```diff
@@ -0,0 +1,4 @@
+; LWOTC Needs Translation
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/LW_FactionBalance/LW_FactionBalance.rus:1]:
```suggestion
; LWOTC Translated
```

File: LongWarOfTheChosen/Localization/LW_FactionBalance/LW_FactionBalance.rus [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+[X2Effect_VoidConduitPatch]
+ActionsLeftFlyoverText="Осталось действий: <XGParam:IntValue0/>"
+; End Translation
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/LW_FactionBalance/LW_FactionBalance.rus:4]:
```suggestion
; End Translated
```

File: LongWarOfTheChosen/Localization/LW_Overhaul.rus [hunk 3]
```diff
@@ -573,6 +578,9 @@ m_strLiberatedCityPrefix="Свободный город"
 [XComGameState_LWOutPost]
 m_strProhibitedJobAlert="Из-за рейда \"Адвента\" в регионе <XGParam:StrValue1/> члены Сопротивления не могут быть назначены на работу \"<XGParam:StrValue0/>\" в ближайшие <XGParam:IntValue0/> дн. Все повстанцы, выполнявшие эту работу, вынуждены перейти в режим конспирации."
 m_strProhibitedJobEnded="Члены Сопротивления в регионе <XGParam:StrValue0/>, занятые работой \"<XGParam:StrValue1/>\", снова продуктивно трудятся."
+;LWOTC needs translated
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/LW_Overhaul.rus:581]:
```suggestion
;LWOTC translated
```

File: LongWarOfTheChosen/Localization/LW_WeaponsAndArmor/XComGame.rus [hunk 4]
```diff
@@ -0,0 +1,51 @@
+; LWOTC Needs Translation
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/LW_WeaponsAndArmor/XComGame.rus:1]:
```suggestion
; LWOTC Translated
```

File: LongWarOfTheChosen/Localization/XComGame.int [hunk 5]
```diff
@@ -402,10 +402,10 @@ LocPromotionPopupText="<Bullet/> Your grenades now pierce (ignore) up to 2 point
 [Bombard_LW X2AbilityTemplate]
 LocFriendlyName="Bombardier"
 ; LWOTC Needs Translation
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/XComGame.int:404]:
```suggestion
; LWOTC Translated
```

File: LongWarOfTheChosen/Localization/XComGame.int [hunk 6]
```diff
@@ -402,10 +402,10 @@ LocPromotionPopupText="<Bullet/> Your grenades now pierce (ignore) up to 2 point
...
-LocLongDescription="You may throw or launch grenades and Claymores two additional tiles."
-LocHelpText="You may throw or launch grenades and Claymores two additional tiles."
+LocLongDescription="You may throw or launch grenades two additional tiles."
+LocHelpText="You may throw or launch grenades two additional tiles."
 LocFlyOverText="Bombardier"
-LocPromotionPopupText="<Bullet/> Your grenades have 2 tiles extra range.<br/><Bullet/> This includes both launched and thrown grenades, as well as Reaper Claymores.<br/>"
+LocPromotionPopupText="<Bullet/> Your grenades have 2 tiles extra range.<br/><Bullet/> This includes both launched and thrown grenades.<br/>"
 ; End Translation
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/XComGame.int:409]:
```suggestion
; End Translated
```

File: LongWarOfTheChosen/Localization/XComGame.rus [hunk 7]
```diff
@@ -101,6 +101,9 @@ RankIcons[6]="UILibrary_Common.rank_colonel"
 RankIcons[7]="UILibrary_Common.rank_commander"
 RankIcons[8]="UILibrary_Common.rank_fieldmarshall"
 
+
+
+
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/XComGame.rus:106]:
```suggestion
```

File: LongWarOfTheChosen/Localization/XComGame.rus [hunk 8]
```diff
@@ -1046,25 +1063,25 @@ LWOTC Translated
...
 FriendlyNamePlural="Элитные приклады"
-BriefSummary="Промахи из основного оружия с <Ability:STOCK_SUP_GF_CHANCE/>% шансом могут стать царапиной."
-TinySummary="Промахи могут царапать."
-LootTooltip="Эта оружейная модификация после установки иногда делает промахи царапинами."
+BriefSummary="Добавляет способность устойчивая позиция с <Ability:STOCK_SUP_GF_CHANCE/>% бонусом к точности и крит.шансу при следующем выстреле, если боец не будет двигаться."
+TinySummary="Дает способность "устойчивая позиция""
+LootTooltip="Эта оружейная модификация после установки добавляет способность устойчивая позиция."
 
-;End Trasnlated
+;End Trasnlated_2
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/XComGame.rus:1084]:
```suggestion
;End Translated
```

File: LongWarOfTheChosen/Localization/XComGame.rus [hunk 9]
```diff
@@ -4844,7 +5339,37 @@ LocLongDescription="Предоставляет дополнительное дв
...
+
+
+; LWOTC Translated
+[BloodTrail X2AbilityTemplate]
+LocFriendlyName="Кровавый след"
+LocLongDescription="Выстрел делает +<Ability:BLOODTRAILDAMAGE/> урона если цель была уже ранена на этом ходу. Добавляет кровотечение, не суммируется."
+LocHelpText=="Выстрел делает +<Ability:BLOODTRAILDAMAGE/> урона если цель была уже ранена на этом ходу. Добавляет кровотечение, не суммируется."
+
+LocPromotionPopupText="<Bullet/> Действует только на тот ход, когда был ранен противник.<br/><Bullet/> Планируйте наперед, чтобы использовать преимущество кровавого следа.<br/><Bullet/> Кровотечение наносит 1 урон в течении 2 ходов.<br/><Bullet/> Последующие выстрелы лишь обновляют время действия кровотечения до 2 ходов.<br/><Bullet/> Кровотечение так же работает и при броске ножей.<br/>"
+; End Translated
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/XComGame.rus:5372]: Some of these [will change shortly](https://github.com/long-war-2/lwotc/pull/1178/files#diff-0c3559cce935fc36aef53dcc259c82aeacc54c87769f5883b5fe3cd0d51b32d2L5235).

**Review by pledbrook — CHANGES_REQUESTED** [review]
Thank you! This was a lot of work and much appreciated. I've just fixed up some markers via GitHub suggestions. You should be able to click on the "Add suggestion to batch" button for each of them and then commit that batch once they're all added.

**Review by pledbrook — APPROVED** [review]
This is great, thanks. I will postpone merging this until the Reaper changes have been merged.

**Pull request merged**